src/sql-parser/src/parser.rs

```diff
@@ -4639,7 +4639,7 @@ impl<'a> Parser<'a> {
         })
     }
 
-    /// Parse an expression, optionally followed by ASC or DESC (used in ORDER BY)
+    /// Parse an expression, optionally followed by ASC or DESC (used in ORDER BY), and then `[NULLS { FIRST | LAST }]`
```

Comment:
Nit: I'd move the "(used in ORDER BY)" to the end, since it applies to both ASC/DESC and NULLS.

Also the line is a bit long for my taste. `rustfmt` doesn't muck with the line length of comments, so you need to take care of keeping that reasonable yourself. I think most people try to keep comments to 80 or 100 characters.

Reply:
done